### PR TITLE
50 funds receiver

### PR DESCRIPTION
**Pull request by andygray**
* [x] deploy receiver clone to mainnet
* [x] add handler 
* [x] set up clone

### CODE REVIEW

File: scripts/11_create_v1_funds_receiver_clone.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+  // 0x9b4E02227952214e1cD4aE60ed757589f2DF9661 RINKEBY
+  const collabRoyaltiesRegistryAddress = prompt(`collabRoyaltiesRegistry deployer address? `);
+  const collabRoyaltiesRegistry = new ethers.Contract(
+    collabRoyaltiesRegistryAddress,
+    CollabRoyaltiesRegistry.abi,
+    deployer
+  );
+
+  // 0xb4bb0960b5095e5a0abd07d18803f45c4c4eadf6 RINKEBY
```
Comment (jamesmorgan) [inline scripts/11_create_v1_funds_receiver_clone.js:76]: Can we add to the `README` and verify any sources on both networks to be consistent with the others contracts

File: scripts/11_create_v1_funds_receiver_clone.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+const twoPercent = '200000';
+
+async function main() {
+  const [deployer] = await ethers.getSigners();
+  console.log('Deploying V1 funds handler with account:', await deployer.getAddress());
+
+  const {name: network} = hre.network;
+  console.log(`Running on network [${network}]`);
+
+  // 0x9b4E02227952214e1cD4aE60ed757589f2DF9661 RINKEBY
```
Comment (jamesmorgan) [inline scripts/11_create_v1_funds_receiver_clone.js:68]: Ditch comment if not useful

File: scripts/10_setup_v1_funds_receiver_clone.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+  // 0x9b4E02227952214e1cD4aE60ed757589f2DF9661 RINKEBY
+  const collabRoyaltiesRegistryAddress = prompt(`collabRoyaltiesRegistry deployer address? `);
+  const collabRoyaltiesRegistry = new ethers.Contract(
+    collabRoyaltiesRegistryAddress,
+    CollabRoyaltiesRegistry.abi,
+    deployer
+  );
+
+  // 0x2e9B9Eea4611163C38F5831fCbdBD113424772f7 RINKEBY
```
Comment (jamesmorgan) [inline scripts/10_setup_v1_funds_receiver_clone.js:21]: Add to README is useful, ditch comment

**Review by jamesmorgan — CHANGES_REQUESTED** [review]
Minor comments

File: scripts/11_create_v1_funds_receiver_clone.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  '0x2E8D42eeC83E5e9dFC3DF007F8CE890197a1d461',
+  '0xea12c9187f434c6d13f541308bbfb1acb9011acb',
+  '0xa76d16e8eb82110a63c21a99c20d1599a5482771',
+  '0xaf405F251329BaF25C4d8F9bD055395a054144d4',
+  '0x63dfddbc040cdbe68d7727b888965303d2fc316d',
+  '0xde29f3D1bDbAb04c8dB83108c5415438a1eC84CE',
+  '0xB9BEA0554B3CA76660712D6B525CBFbc101fEC1d',
+  '0x79cde9Bb44b079466e4f7f82A2c45bc3924c6Dc8',
+  '0xe3da86b69b74e13e4c45b142989e498745a2e7bf',
+];
```
Comment (vince0656) [inline scripts/11_create_v1_funds_receiver_clone.js:57]: I would use `getAddress` from ethers to make sure these are all valid addresses

Comment (vince0656) [reply scripts/11_create_v1_funds_receiver_clone.js:57]: get address will either return the checksummed address or blow up so needs to be done in a try catch and in the catch need to stop the script running to show that an address is invalid

File: scripts/11_create_v1_funds_receiver_clone.js [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+const twoPercent = '200000';
```
Comment (vince0656) [inline scripts/11_create_v1_funds_receiver_clone.js:59]: whats the decimal places? just checking decimal places are correct

File: scripts/11_create_v1_funds_receiver_clone.js [hunk 6]
```diff
@@ -0,0 +1,94 @@
+const prompt = require('prompt-sync')();
```
Comment (vince0656) [inline scripts/11_create_v1_funds_receiver_clone.js:1]: I think you will find coming back to the file confusing based on the name. This file should be called something like 'setup_collab'

**Review by vince0656 — COMMENTED** [review]
looks great

File: scripts/11_create_v1_funds_receiver_clone.js (hunk 5 above)
Comment (andygray) [reply scripts/11_create_v1_funds_receiver_clone.js:59]: I checked on rinkeby...

**Pull request merged**